ns/math.n3

```diff
@@ -0,0 +1,260 @@
+    rdfs:subClassOf rdf:Property;
+    rdfs:comment "The class of things that compare values.".
+
+math:Value a rdfs:Class; rdfs:label "Value"; 
+   rdfs:subClassOf [owl:unionOf (xsd:decimal xsd:float xsd:double)]; 
+   rdfs:comment "The class of things that derive from XSD numeric datatypes." .
+
+math:Function a rdfs:Class;
+   rdfs:subClassOf owl:FunctionalProperty;
+   rdfs:label "Function";
+   rdfs:comment "A math:Function is unique in terms of math:EqualTo. ".
+
+math:ReverseFunction a rdfs:Class;
+   rdfs:subClassOf owl:InverseFunctionalProperty;
+   rdfs:comment "A math:ReverseFunction is unambiguous in terms of math:EqualTo. ".
+
+math:LogicalOperator a rdfs:Class; rdfs:subClassOf math:Function, math:ReverseFunction;
+   rdfs:comment """A logical operator allows evaluation eihter way, or testing relationship
+between two values""".
+
+math:List a rdfs:Class; rdfs:label "List"; 
+   rdfs:subClassOf log:List; 
+   rdfs:comment """The class of things that are N3 lists where all of the 
+members are math:Value items.""".
+
+math:TwoMemberedList a rdfs:Class; rdfs:label "TwoMemberedList"; 
+   rdfs:subClassOf math:List; 
+   rdfs:comment "This is the class of things that are math lists with only two members." .
+
+math:memberCount a math:Function; rdfs:label "memberCount"; 
+   rdfs:domain math:List; rdfs:range math:Value; 
+   rdfs:comment """The number of items in a list. The subject is a list,
+the object is calculated as the number of members.
+Obsolete, see list:length""";
+   vs:term_status "deprecated".
+
+
+# Logical operators:
+
+math:greaterThan a math:StrictProperty; rdfs:label "greaterThan"; 
+   rdfs:comment """True iff the subject is a number which is greater than the object.""";
+   rdfs:domain math:Value; rdfs:range math:Value .
+
+math:notGreaterThan a math:StrictProperty;
+    rdfs:label "notGreaterThan"; 
+    rdfs:comment """True iff the subject is a number which is NOT greater than the object.""";
+    rdfs:domain math:Value; rdfs:range math:Value .
+
+math:lessThan a math:StrictProperty; rdfs:label "lessThan"; 
+   rdfs:comment """True iff the subject is a number which is LESS than a object.""";
+   rdfs:domain math:Value; rdfs:range math:Value .
+
+math:notLessThan a math:StrictProperty; rdfs:label "notlessThan"; 
+   rdfs:comment """True iff the subject is a number which is NOT LESS than a object.""";
+   rdfs:domain math:Value; rdfs:range math:Value .
+
+math:equalTo a math:StrictProperty; rdfs:label "equalTo"; 
+   rdfs:comment """
+- schema
+$a1 math:equalTo $a2
+- summary
+checks equality of numbers
+- definition
+true if and only if $a1 is equal to $a2. 
+requires both arguments to be either concrete numerals, or variables bound to a numeral.
+- literal domains
+$a1: xs:decimal (or its derived types), xs:float, or xs:double  (see note on type promotion, and casting from string)
+$a2: xs:decimal (or its derived types), xs:float, or xs:double  (see note on type promotion, and casting from string)
```

Comment:
Where is this note on type promotion and casting?

Reply:
This is very much still a work in progress:
https://docs.google.com/document/d/19LSGC7OgGK8oGdPTTrtkgQzYTU5Gq5R-v5FTHf2sHEQ/edit